scripts/generate_default_media_types.py

```diff
@@ -8,7 +8,7 @@
 from pathlib import Path
 
 module_dir = Path(__file__).parent.resolve()
-media_types_py = module_dir / ".." / "whitenoise" / "media_types.py"
+media_types_py = module_dir / "../src/whitenoise/media_types.py"
```

Comment:
Can we use `parent` here instead of `..` and maintain the pathlib way to concatenate string?

Reply:
I believe pathlib internally translates `".."` and `"/"` to equivalent calls, regardless of OS. But if you care enough to make a PR be my guest.